bionty/base/__init__.py

```diff
@@ -42,8 +42,6 @@
 Base class
 ----------
 
-`Pronto Ontology objects <https://pronto.readthedocs.io/en/stable/api/pronto.Ontology.html>`__ can be accessed via `{entity}.to_pronto()`.
-
```

Comment:
is `to_pronto()` still documented somewhere?

Reply:
It's still in the `adding new ontologies` guide I think. But I don't think that users need to know about it, right?

Reply:
I think that's very unlikely but I'll ensure that this is documented somewhere.